### ISSUE TITLE
show bgp all neighbors <neighbor> advertised-routes not working

### ISSUE THREAD

**Issue by mupchu** [CiscoTestAutomation/genieparser#472]
When trying to parse the command 'show bgp all neighbors <neighbor> advertised routes, I get the following error:

fatal: [192.168.86.201]: FAILED! => {"msg": "Unable to parse output for command 'show bgp all neighbors 192.168.100.5 advertised-routes' ('Device' object has no attribute 'execute')"}
fatal: [192.168.86.202]: FAILED! => {"msg": "Unable to parse output for command 'show bgp all neighbors 192.168.100.5 advertised-routes' ('Device' object has no attribute 'execute')"}

does not work when called in ansible directly, or just calling the parser.

**Comment by danielgraziano** [comment]
@mupchu did you connect to the device first? The execute attribute is added once connected

**Comment by mupchu** [comment]
yes, I ran this via ansible, and ran the command, error was when it was parsed.

this was the script:
---
- name: Sh BGP Advertised-Routes direct filter
  hosts: ios-router
  connection: network_cli
  gather_facts: no
  roles:
    - ansible-pyats
  tasks:
    - name: Run command
      cli_command:
        command: show ip bgp all neighbors 192.168.100.5 advertised-routes
      register: cli_output
    
    - name: Parsing output
      set_fact:
        parsed_output: "{{ cli_output.stdout | pyats_parser('show bgp all neighbors 192.168.100.5 advertised-routes', 'ios') }}"

**Comment by domachad** [comment]
Hi @mupchu,

It doesn't seem like there is a problem with the parser. 
Could there be something missing in your script?
Did you try executing a different command?

**Comment by mupchu** [comment]
I tried running similar command and it did work.  I tried running the command directly, and tried running the command and then calling the parser on the output.  Both failed for 'show ip bgp all neighbors <neighbor> advertised routes'.  I am attaching both the output from ansible when I tried running the plays, and the plays themselves (combined in a txt file).  I also ran similar command on nxos device and that did work, but most of our routers are IOS based.

This was done in a lab as proof of concept so I can build out linux servers to run ansible and pyats on.  
[playbooks.txt](https://github.com/CiscoTestAutomation/genieparser/files/6679419/playbooks.txt)
[ansible output.txt](https://github.com/CiscoTestAutomation/genieparser/files/6679420/ansible.output.txt)
